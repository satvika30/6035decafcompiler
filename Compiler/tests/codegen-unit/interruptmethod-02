class Program {
	int a[10];
	
	void main() {
		a[0] = 1;
		callout("printf","Handler allowed correct access\n");
		
		callout("printf","Checking greater than length index\n");
		a[11] = 0;
		
		callout("printf","Shit! Not working\n");
	}
}