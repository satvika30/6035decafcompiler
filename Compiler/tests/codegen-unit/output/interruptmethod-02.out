Handler allowed correct access
Checking greater than length index
Array index out of bounds (9, 17)